lms/djangoapps/discussion_api/tests/test_api.py

```diff
@@ -131,6 +131,20 @@ def test_basic(self):
             }
         )
 
+
+@attr('shard_2')
+@ddt.ddt
```

Comment:
nit: You can now take the ddt decorator off the GetCourseTest class up on line 88.